src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,28 @@ public override SwitchParameter UseSSL
             }
         }
 
+        /// <summary>
+        /// Gets or sets strict mode.
+        /// </summary>
+        [Experimental(ExperimentalFeature.PSStrictModeAssignment, ExperimentAction.Show)]
```

Comment:
`ExperimentalFeature.PSStrictModeAssignement` doesn't currently exist.
Please add `internal const string PSStrictModeAssignment = "PSStrictModeAssigment";` to line 31 in `ExperimentalFeature.cs` file above.